packages/utilities/src/osDetector.ts

```diff
@@ -6,7 +8,9 @@ let isMacResult: boolean | undefined;
  */
 export function isMac(reset?: boolean): boolean {
   if (typeof isMacResult === 'undefined' || reset) {
-    const userAgent = typeof window !== 'undefined' && window.navigator.userAgent;
+    const win = getWindow();
+    const userAgent = win && window.navigator.userAgent;
```

Comment:
```suggestion
    const userAgent = win && win.navigator.userAgent;
```

Don't you mean `win && win.nav...`?

Reply:
nice catch, fixed